pypesto/visualize/waterfall.py

```diff
@@ -256,8 +259,8 @@ def get_fvals(result: Result,
     # reduce to indices for which the user asked
     fvals = fvals[start_indices]
 
-    # get the minimal value which should be plotted
-    min_val = np.min(fvals)
+    # get the minimal value which should be plotted, avoid the value being -inf or nan
+    min_val = np.nanmin(fvals[fvals != -np.inf])
```

Comment:
is this necessary? nans+infs should have been removed already?

Reply:
Agree with Yannik. `nanmin ` should not be necessary any more...

Reply:
but if it is -inf, then min(fvals) will be -inf

Reply:
but -np.inf should also not happen, right? nan+ plus/minus infs have been removed in delete_nan_inf?

Reply:
ah ok sorry, you are right (I think). delete_nan_inf is not called before in this function.